Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -4434,67 +4433,67 @@
     ]
   },
   {
-    "Input": "Ya terminé todos los trabajos hace más de 2 semanas",
-    "NotSupported": "dotnet, java",
+    "Input": "durante más de 2 semanas antes de hoy",
+    "NotSupported": "java",
     "NotSupportedByDesign": "javascript, python",
     "Results": [
       {
-        "Text": "hace más de 2 semanas",
+        "Text": "durante más de 2 semanas antes de hoy",
         "Type": "daterange",
-        "Start": 30,
-        "Length": 21
+        "Start": 0,
+        "Length": 37
       }
     ]
   },
   {
-    "Input": "Volveré dentro de 2 semanas",
-    "NotSupported": "dotnet, java",
+    "Input": "dentro de 2 semanas a partir de hoy",
+    "NotSupported": "java",
     "NotSupportedByDesign": "javascript, python",
     "Results": [
       {
-        "Text": "dentro de 2 semanas",
+        "Text": "dentro de 2 semanas a partir de hoy",
         "Type": "daterange",
-        "Start": 8,
-        "Length": 19
+        "Start": 0,
+        "Length": 35
       }
     ]
   },
   {
-    "Input": "Volveré dentro de menos de 2 semanas",
-    "NotSupported": "dotnet, java",
+    "Input": "dentro de menos de 2 semanas a partir de hoy",
+    "NotSupported": "java",
     "NotSupportedByDesign": "javascript, python",
     "Results": [
       {
-        "Text": "dentro de menos de 2 semanas",
+        "Text": "dentro de menos de 2 semanas a partir de hoy",
         "Type": "daterange",
-        "Start": 8,
-        "Length": 28
+        "Start": 0,
+        "Length": 44
       }
     ]
   },
   {
-    "Input": "Esta tarea debió ser terminada más de tres días antes",
-    "NotSupported": "dotnet, java",
+    "Input": "durante más de 2 días antes de ayer",
```

Comment:
Same. Please don't modify specs unless they are really incorrect.
Feel free to add as many new ones as possible though. In this case here, both the written numbers (e.g., "tres") or the numerals (e.g., "2") should be supported.

Reply:
will add specs, thanks